src/index.html

```diff
@@ -5,10 +5,158 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Dia</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Dia</h1>
+    <header class="header">
+      <nav class="nav header__nav">
+        <img src="images/logo/Air.png" alt="logo" class="logo header__logo">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="/" class="nav__link">Home</a></li>
```

Comment:
не пиши все в один рядок
```suggestion
          <li class="nav__item">
            <a href="/" class="nav__link">Home</a>
          </li>
```